File: warmups/4jantue/warmup_sum_of_multiples.js
const sumOfMultiples = (numInput) => {
  var totalSum = 0;
  for (let i = 0; i <= numInput; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      totalSum += i;
    }
  }
  return totalSum;
};

sumOfMultiples(10); //=> 23
sumOfMultiples(16); //=> 60

File: warmups/4jantue/warmup_sum_of_multiples.rb
def sum_of_multiples (num_input)
    total_sum = 0
    i = 0
    while i < num_input 
        if i % 3 == 0 || i % 5 == 0
           total_sum += i 
        end
        i += 1
    end
    return total_sum;

end

p sum_of_multiples(10) #=>23
p sum_of_multiples(16) #=>60

